.github/workflows/deploy.yml

```diff
@@ -29,3 +29,20 @@ jobs:
         sudo apt-get install moreutils
     - name: deploy backend
       run: make ci-deploy -C browser/backend/chalice
+
+  frontend:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Configure AWS Credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-region: us-east-1
```

Comment:
Just curious, why doesn't the `single-cell-dev` profile need to be configured to deploy?

Reply:
Oh right, you create a policy in terraform :+1:

Reply:
the single-cell-dev profile is an alias for all of the above configuration. We don't actually require an AWS_PROFILE to be named single-cell-dev anywhere.

Reply:
The above configuration are the account details i.e. stored in `.aws/credentials` . `single-cell-dev` is a profile for an IAM role that I don't believe is reflected here.